postgres/pg-stack.yml

```diff
@@ -3,7 +3,7 @@ version: '3.1'
 
 services:
   db:
-    image: postgres:15-alpine
+    image: postgres:16-alpine
```

Comment:
I you can see in this PR it is not always compatible to take the latest version. So please keep the exact version in the PRs. To keep the builds reproducible.

Reply:
Additionally in production they latest version is still 13. So there is no rush to test using the latest and greatest version.

Reply:
Also fine for me.
I ask myself whether we should use the "latest" version on BTP instead?

Reply:
I do think that the productive Postgres version should be tested. Whether that is in additional to the latest version or instead of the latest version is both fine with me. The advantage of also testing the latest version would give a timely heads up for things breaking in the future (around 2-3 years warning period).

Reply:
Based on the documentation page for [Parameters](https://help.sap.com/docs/postgresql-hyperscaler-option/postgresql-on-sap-btp-hyperscaler-option/parameters?locale=en-US&q=engine_version) for the PostgreSQL on SAP BTP, Hyperscaler Option, the latest available version is 14. So I would think the tests should run with 14, 15 and 16.